docs/content/docs/internals/job_scheduling.md

```diff
@@ -95,3 +95,22 @@ For that reason, the execution of an ExecutionVertex is tracked in an {{< gh_lin
 {{< img src="/fig/state_machine.svg" alt="States and Transitions of Task Executions" width="50%" >}}
 
 {{< top >}}
+
+## Repeatable Resource Cleanup Strategy
+
+Once a job has reached a terminally global state of either finished, failed or cancelled, the
```

Comment:
```suggestion
Once a job has reached a globally terminal state of either finished, failed or cancelled, the
```
"globally terminal" is the usual phrasing as far as I remember

Reply:
Yep, you're right. I think i've just typed the phrase "globally terminal" so much my brain switched the two around to liven things up.